src/shared/report_op.c

```diff
@@ -195,7 +195,7 @@ static int _report_filter_value(const char *filter_by, int prev_filter)
         return (prev_filter);
     } else {
         merror("Invalid relation '%s'.", filter_by);
-        return (-1);
+        return -1;
```

Comment:
The rest of return calls for this function is written with brackets, change them to be consistent.